actions.js

```diff
@@ -77,6 +77,16 @@ module.exports = {
 			}
 		}
 
+		if (MODEL_ACTIONS?.onScreenMenu) {
+			actions['onScreenMenu'] = {
+				label: MODEL_ACTIONS.onScreenMenu.name,
+				callback: (action) => {
+					cmd = VISCA.MSG_OPERATION + VISCA.CAM_MENU + VISCA.DATA_TOGGLEVAL + VISCA.END_MSG
```

Comment:
It look as though there is an undocumented API call for this.. ... 
/birddogptz 
{"Menu":"On/Off")